typescript/utils/src/altvm.ts

```diff
@@ -0,0 +1,657 @@
+import { MINIMUM_GAS } from './mingas.js';
+import { ProtocolType } from './types.js';
+
+// ### QUERY BASE ###
+export type ReqGetBalance = { address: string; denom: string };
+
+export type ReqGetTotalSupply = { denom: string };
+
+export type ReqEstimateTransactionFee<T> = {
+  transaction: T;
+  estimatedGasPrice: string;
+  sender: string;
+  senderPubKey?: string;
+  memo?: string;
```

Comment:
i think memo is something cosmos specific, shouldn't this be part of the provided transaction data instead?